### PR TITLE
Fix NPE when OIDC client has no redirect URIs defined

### PR DESCRIPTION
**Pull request by skybert**

### CODE REVIEW

File: Server/src/main/java/org/xdi/oxauth/service/RedirectionUriService.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public String validateRedirectionUri(String clientIdentifier, String redirection
 
                 if (StringUtils.isNotBlank(redirectionUri)) {
                     log.debug("Validating redirection URI: clientIdentifier = {}, redirectionUri = {}, found = {}",
-                            clientIdentifier, redirectionUri, redirectUris.length);
+                            clientIdentifier, redirectionUri, redirectUris != null ? redirectUris.length : 0);
 
                     final String redirectUriWithoutParams = uriWithoutParams(redirectionUri);
```
Comment (yuriyz) [inline Server/src/main/java/org/xdi/oxauth/service/RedirectionUriService.java]: Here in line 81 we have `redirectUris` in for loop. If it is null then in line 81 we will get NPE. It means we have to put null check somewhere above and handle it correctly.

Comment (skybert) [reply Server/src/main/java/org/xdi/oxauth/service/RedirectionUriService.java]: You're right. I've put the `null` guard in the enclosing `if`, have a look in d217dd80cdd5f8ce597bfc2e383451ca4b1a9b81

**Comment by skybert** [comment]
@yuriyz : I've put the null guard in the enclosing if, have a look in `d217dd8` and updated the PR.

**Comment by yuriyz** [comment]
@skybert merged, thanks.

**Pull request merged**